frontend/src/actions/paginationActions.js

```diff
@@ -102,3 +108,7 @@ export const checkTableData = (newPage, currType) => (dispatch, getState) => {
     dispatch(fetchActions[currType]());
   }
 };
+
+export const fetchNextJobs = (newPage) => (dispatch) => {
+  dispatch(checkIlabJobs(newPage));
+};
```

Comment:
I suspect this is also old code that got merged back in. It's referenced in one place later, where it's diagnosed as "imported but unused".